var/spack/repos/builtin/packages/amrex/package.py

```diff
@@ -115,6 +115,7 @@ class Amrex(CMakePackage, CudaPackage, ROCmPackage):
     depends_on('cmake@3.14:', type='build', when='@19.04:')
     # cmake @3.17: is necessary to handle cuda @11: correctly
     depends_on('cmake@3.17:', type='build', when='^cuda @11:')
+    depends_on('cmake@3.17:', type='build', when='@22.07:')
```

Comment:
Actually already in `22.06`: https://github.com/AMReX-Codes/amrex/pull/2757
```suggestion
    depends_on('cmake@3.17:', type='build', when='@22.06:')
```